analyzer/library/pip/pip_test.go

```diff
@@ -0,0 +1,102 @@
+package pip
+
+import (
+	"os"
+	"sort"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/aquasecurity/fanal/analyzer"
+	"github.com/aquasecurity/fanal/types"
+	godeptypes "github.com/aquasecurity/go-dep-parser/pkg/types"
+)
+
+func Test_gomodAnalyzer_Analyze(t *testing.T) {
+	tests := []struct {
+		name      string
+		inputFile string
+		want      *analyzer.AnalysisResult
+		wantErr   string
+	}{
+		{
+			name:      "happy path",
+			inputFile: "testdata/requirements.txt",
+			want: &analyzer.AnalysisResult{
+				Applications: []types.Application{
+					{
+						Type:     types.Pip,
+						FilePath: "testdata/requirements.txt",
+						Libraries: []types.LibraryInfo{
+							{Library: godeptypes.Library{Name: "click", Version: "8.0.0"}},
+							{Library: godeptypes.Library{Name: "Flask", Version: "2.0.0"}},
+							{Library: godeptypes.Library{Name: "itsdangerous", Version: "2.0.0"}},
+							{Library: godeptypes.Library{Name: "Jinja2", Version: "3.0.0"}},
+							{Library: godeptypes.Library{Name: "MarkupSafe", Version: "2.0.0"}},
+							{Library: godeptypes.Library{Name: "Werkzeug", Version: "2.0.0"}},
+						},
+					},
+				},
+			},
+		}, {
+			name:      "sad path",
+			inputFile: "testdata/invalid.txt",
+			want:      nil,
```

Comment:
It is not a sad path. It means some errors occur.

Reply:
renamed, didn't find real `sad`  path